users/tests.py

```diff
@@ -0,0 +1,23 @@
+from django.test import TestCase
+from users.models import User
+
+import faker
+
+class UserTestCase(TestCase):
+  def setUp(self):
+        self.fake=faker.Faker()
+        self.first_name=self.fake.first_name()
+        self.last_name=self.fake.last_name()
+        self.email=self.fake.email()
+        self.gender='NS'
+
+        User.objects.create(first_name=self.first_name, last_name=self.last_name,
+                            email=self.email, gender=self.gender)
```

Comment:
We follow the "[hanging indentation](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#index-3:~:text=Use%20four%20spaces%20for%20indentation.,'shortened%20for%20clarity)" style used by Django instead of the vertical alignment. Also mentioned in the [tech guides](https://github.com/rootstrap/tech-guides/tree/master/python#indentation)

```suggestion
        User.objects.create(
            first_name=self.first_name,
            last_name=self.last_name,
            email=self.email,
            gender=self.gender,
        )
```